core-api/src/main/java/com/optimizely/ab/decision/bucketer/DecisionBucketer.java

```diff
@@ -0,0 +1,43 @@
+/****************************************************************************
+ * Copyright 2019, Optimizely, Inc. and contributors                        *
+ *                                                                          *
+ * Licensed under the Apache License, Version 2.0 (the "License");          *
+ * you may not use this file except in compliance with the License.         *
+ * You may obtain a copy of the License at                                  *
+ *                                                                          *
+ *    http://www.apache.org/licenses/LICENSE-2.0                            *
+ *                                                                          *
+ * Unless required by applicable law or agreed to in writing, software      *
+ * distributed under the License is distributed on an "AS IS" BASIS,        *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. *
+ * See the License for the specific language governing permissions and      *
+ * limitations under the License.                                           *
+ ***************************************************************************/
+package com.optimizely.ab.decision.bucketer;
+
+import com.optimizely.ab.annotations.VisibleForTesting;
+
+/**
+ Generate bucket value with in the provided range
```

Comment:
Can you please copy the comments from the java master. 
```/**
 * Default Optimizely bucketing algorithm that evenly distributes users using the Murmur3 hash of some provided
 * identifier.
 * <p>
 * The user identifier <i>must</i> be provided in the first data argument passed to
 * {@link #bucket(Experiment, String, ProjectConfig)} and <i>must</i> be non-null and non-empty.
 *
 * @see <a href="https://en.wikipedia.org/wiki/MurmurHash">MurmurHash</a>
 */```
Make sure it's only for implementation classes, not for interface.

Reply:
It couldn't be copied completely, as it doesn't have bucket method now and its docs is still describing it.